fxevent/console_test.go

```diff
@@ -131,7 +133,7 @@ func TestConsoleLogger(t *testing.T) {
 		{
 			name: "Invoking",
 			give: &Invoking{FunctionName: "bytes.NewBuffer()"},
-			want: "[Fx] INVOKE		bytes.NewBuffer()\n",
+			want: "[Fx] INVOKE		bytes.NewBuffer() from Module \"\"\n",
```

Comment:
I think maybe we should omit the "from module ..." if module is empty.

(fx.New is empty, right? That'll help differentiate top-level provide/invoke/decorates from module-level.)